lld/test/ELF/eh-frame-type.test

```diff
@@ -1,5 +1,8 @@
 ## Show that LLD can handle .eh_frame sections of different types.
 
+## Temporary disabled for Windows, fix is required for ld.lld non-zero exit code
```

Comment:
This file, unlike the following ones, seems to exist upstream; so your observation may be a valuable contribution to the LLVM community.  (Though the maintainers might not react diligently to a documentation PR.)  OTOH, it might be something specific to us.  So I think that either this comment should be wrapped in Matter Labs’ Begin/End markers, and/or it should be upstreamed so that we don’t have to do that.